invenio/modules/oauth2server/models.py

```diff
@@ -167,7 +168,7 @@ class Client(db.Model):
         default=u'',
     )
 
-    user_id = db.Column(db.ForeignKey('user.id'))
+    user_id = db.Column(db.ForeignKey('user.id'), nullable=True)
```

Comment:
That's what I wanted to do.. but then, I checked the db before and after... and, surprisingly, it's always `DEFAULT NULL`.